cmd/zetaclientd/start.go

```diff
@@ -204,8 +206,19 @@ func start(_ *cobra.Command, _ []string) error {
 	}
 
 	telemetryServer.SetIPAddress(cfg.PublicIP)
+
+	nodeAccounts := appContext.GetNodeAccounts()
+	whitelistedPeers := []peer.ID{}
+	for _, nodeAccount := range nodeAccounts {
+		pid, err := conversion.Bech32PubkeyToPeerID(nodeAccount.GranteePubkey.Secp256k1.String())
+		if err != nil {
+			return err
```

Comment:
nit/question: should we return an error here or simply `continue`? Do we need all the `nodeAccounts` to start the chain?